pkg/webui/lib/render-callback.js

```diff
@@ -12,6 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-.title
-  text-margin-top($ls.m)
-  text-margin-bottom($ls.m)
+const renderCallback = ({ render, children }, context) => {
```

Comment:
Please add JsDoc, it took me a bit to understand the usage in the bigger context.

Reply:
Also note that the header says `Copyright 2019`.